common/service-dialog-creation-using-dialog-editor.adoc

```diff
@@ -1,11 +1,11 @@
-{product-title} adds a new drag-and-drop service dialog editor. The editor, with its drag-and-drop feature, provides a visual representation of the components that comprise a service dialog - dialog tabs, boxes, and elements - to help you easily use them to design your dialog.
+{product-title} adds a new drag-and-drop service dialog editor to create service dialogs. The editor, with its drag-and-drop feature, provides a visual representation of the components that comprise a service dialog, that is dialog tabs, sections (previously referred as boxes), and elements to help you easily use them to design your dialog.
 
-Using a service dialog, you can determine which options the users get to change. When users access a service, the majority of options available to them are preset and cannot be altered. A service dialog exposes some of those options to the user so that even if they are ordering a basic Red Hat Enterprise Linux 7 machine, for example, they can at least choose the amount of memory, virtual CPUs, or other options available to the instance they order. In cases where certain fields must be unique, such as the name of virtual machines in Red Hat Virtualization, users must enter their own unique name for the virtual machine they choose or the operation will fail, so this field must be exposed.
+Using a service dialog, you can determine what options users get to change. When users access a service, the majority of options available to them are preset and cannot be altered. A service dialog exposes some of those options to the user so that even if they are ordering a basic Red Hat Enterprise Linux 7 machine, for example, they can at least choose the amount of memory, virtual CPUs, or other options available to the instance they order. In cases where certain fields must be unique, such as the name of virtual machines in Red Hat Virtualization, users must enter their own unique name for the virtual machine they choose or the operation will fail, so this field must be exposed.
```

Comment:
I would recommend some revision to this paragraph. Perhaps restructure to state that 'options and fields should be determined based on requirements for the service ordered. Then I'd utlize each of the examples you've submitted. In case #1 (RHEL) users may want to determine a, b, c. Then case #2, where options are more of a hard requirement as they impact the success of an ordered service. 
Let me know if you agree.

Reply:
Chris, thanks for the suggestion. I've updated the text a bit as well as referenced the example in the appendix (unable to xref the example at the moment as the other PR #578 is yet to be merged).